manifests/init.pp

```diff
@@ -30,7 +30,7 @@
   if $facts.dig('pe_databases', 'have_systemd') {
     if $manage_database_maintenance and (versioncmp('2019.0.2', $facts['pe_server_version']) <= 0) {
       class {'pe_databases::pg_repack':
-        disable_maintenance => lookup('pe_databases::maintenance::disable_maintenance', {'default_value' => false}),
+        disable_maintenance => lookup('pe_databases::disable_maintenance', {'default_value' => false}),
```

Comment:
What does the `$manage_database_maintenance` parameter do from line 7? Can we move this lookup to that parameter?

Reply:
yeah the lookup seems like an extra step, can we just do something like `maintenance_ensure = 'present'` and set disable_maintenance (or change that naming also) in the pe_databases::pg_repack based on the parameter and not the lookup

Reply:
I'm not sure it was written this way originally.  Looking at it again, I think it had something to do with disabling the `vacuum` class when enabling the `repack` class.  Since we don't do that anymore, I think it makes sense to just yank out this `disable_maintenance` part.  To me it makes sense to have this determined by `pe_databases::manage_database_maintenance` alone.

Reply:
Alright my previous comment is totally wrong.  We need to keep the parameter at least for backwards compatibility, but also because we need a way to include the repack class and manage absent/ensure on the services and timers.  So I think moving the lookup to a _new_ parameter makes sense.  I'll test that out and push a commit here if it works.